hnn_core/cell.py

```diff
@@ -189,11 +189,6 @@ class Cell:
     ----------
     pos : list of length 3
         The position of the cell.
-    sections : dict
-        The sections. The key is the name of the section
-        and the value is an instance of h.Section.
-    synapses : dict
-        The synapses that the cell can use for connections.
```

Comment:
Why were these removed from the docstring? They're still attributes right?

Reply:
Good catch! Fixed now